app/views/resources/Javascripts.scala.html

```diff
@@ -17,6 +17,3 @@
 @this(autocompleteJavascript: HmrcAccessibleAutocompleteJavascript)
 
 @()(implicit request: Request[_])
-
-<script src='@routes.Assets.versioned("javascripts/application.js")' @{CSPNonce.attr}></script>
-@autocompleteJavascript()
```

Comment:
We still need `@autocompleteJavascript()` - this provides important functionality to the dropdown component